Firebase/Auth/Source/FIRAuth_Internal.h

```diff
@@ -124,6 +150,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)signOutByForceWithUserID:(NSString *)userID error:(NSError *_Nullable *_Nullable)error;
 
+/** @fn operationType
+    @param operationType The value of the FIRAuthOperationType enum which will be translated to its
+        corresponding string value.
+    @return The string value corresponding to the FIRAuthOperationType argument.
+ */
+NSString *const operationType(FIRAuthOperationType operationType);
```

Comment:
nit: If this is going to be exposed in a header file (albeit a private one), it should follow Cocoa naming conventions (e.g. `FIRStringFromOperationType()`).

Reply:
Done.